tests/core/test_accounts.py

```diff
@@ -97,6 +99,22 @@ def acct(request):
     return Account
 
 
+def test_eth_account_default_kdf():
+    assert eth_account.account.defaultKDF == 'scrypt'
+
+
+def test_eth_account_default_kdf_from_env():
+    import importlib
+    os.environ['ETH_ACCOUNT_KDF'] = 'pbkdf2'
```

Comment:
For this type of thing you should use the `monkeypatch` fixture provided by pytest

https://docs.pytest.org/en/latest/monkeypatch.html

Allows you to modify environment variables in a way that they are always cleaned up.

Reply:
Weirdly the `setenv` method disappeared from recent docs. Here's an old copy with the reference: https://docs.pytest.org/en/3.0.0/monkeypatch.html

Reply:
Updated the tests, but refer to this comment:
https://github.com/ethereum/eth-account/pull/38#discussion_r224943961